backend/src/main/java/wooteco/studylog/log/web/controller/StudyLogController.java

```diff
@@ -0,0 +1,87 @@
+package wooteco.studylog.log.web.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.studylog.log.service.StudyLogService;
+import wooteco.studylog.log.web.controller.dto.*;
+
+import java.net.URI;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@RestController
+@RequestMapping("/logs")
+public class StudyLogController {
+    private final StudyLogService studyLogService;
+
+    public StudyLogController(StudyLogService studyLogService) {
+        this.studyLogService = studyLogService;
+    }
+
+    @GetMapping
+    public ResponseEntity<LogResponses> showAll() {
+        List<LogResponse> rowLogResponses = Collections.singletonList(
+                new LogResponse(1L,
+                        new AuthorResponse(1L, "뽀모", "image"),
+                        LocalDateTime.now(),
+                        new CategoryResponse(1L, "미션1"),
+                        "제목",
+                        "내용",
+                        Arrays.asList("자바", "쟈스")
+                )
+        );
+        LogResponses studyLogResponses = new LogResponses(rowLogResponses);
+        return ResponseEntity.ok(studyLogResponses);
+//        return ResponseEntity.ok(studyLogService.findAll());
+    }
+
+    @PostMapping
+    public ResponseEntity<LogResponse> createLog(@RequestBody List<LogRequest> logRequest) {
+        LogResponse logResponse = new LogResponse(1L,
+                new AuthorResponse(1L, "뽀모", "image"),
+                LocalDateTime.now(),
+                new CategoryResponse(1L, "미션1"),
+                "제목",
+                "내용",
+                Arrays.asList("자바", "쟈스"));
+        studyLogService.insertLogs(logRequest);
+        return ResponseEntity.created(URI.create("/logs/" + logResponse.getLogId())).body(logResponse);
+//        LogResponse logResponse = studyLogService.insertLog(
+//                logRequest.getCategoryId(),
+//                logRequest.getTitle(),
+//                logRequest.getTags(),
+//                logRequest.getContent());
+//        return ResponseEntity.created(URI.create("/logs/" + logResponse.getLogId())).body(logResponse);
+    }
+
+    @GetMapping("/{logId}")
+    public ResponseEntity<LogResponse> showLog(@PathVariable Long logId){
```

Comment:
전체 path가 `/logs/{logId}`가 될 것 같은데
logs 이후에 오는 값은 log의 id임을 유추할 수 있으므로 id만 표기하는건 어떤가요?

```suggestion
    @GetMapping("/{id}")
    public ResponseEntity<LogResponse> showLog(@PathVariable Long id){
```